src/wallet/rpcwallet.cpp

```diff
@@ -1101,6 +1101,38 @@ static UniValue getbalance(const JSONRPCRequest& request)
     return ValueFromAmount(pwallet->GetBalance(filter, min_depth));
 }
 
+static UniValue getspendablebalance(const JSONRPCRequest& request){
+    std::shared_ptr<CWallet> const wallet = GetWalletForJSONRPCRequest(request);
+    CWallet *const pwallet = wallet.get();
+
+    if (!EnsureWalletIsAvailable(pwallet, request.fHelp)) {
+        return NullUniValue;
+    }
+
+    if (request.fHelp || (request.params.size() > 0 )){
+        throw std::runtime_error(
+            std::string(
+                "getSpendableBalance ()\n"
```

Comment:
this name should be the same as the rpc call. No need to make it be camelcase

Reply:
Will get this changed now.

Thanks.

Reply:
Updated and squashed.